sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/SdkVersionUtils.cs

```diff
@@ -11,7 +11,7 @@ namespace Azure.Monitor.OpenTelemetry.Exporter
 {
     internal static class SdkVersionUtils
     {
-        internal static string SdkVersion = GetSdkVersion();
+        internal static string s_sdkVersion = GetSdkVersion();
```

Comment:
Curios to know why we the variables are starting with `s_`.

Reply:
static variables start with s_